.github/workflows/nightlies.yml

```diff
@@ -63,3 +63,23 @@ jobs:
             --user scipy-wheels-nightly \
             --skip-existing \
             dist/matplotlib-*.whl
+
+      - name: Remove old uploads to save space
+        shell: bash
+        run: |
+          N_LATEST_UPLOADS=5
+
+          # Remove all _but_ the last "${N_LATEST_UPLOADS}" package versions
+          # N.B.: `anaconda show` places the newest packages at the bottom of the output
+          # of the 'Versions' section and package versions are preceded with a '   + '.
+          anaconda show scipy-wheels-nightly/matplotlib &> >(grep '+') | \
+              sed 's/.* + //' | \
+              head --lines "-${N_LATEST_UPLOADS}" > remove-package-versions.txt
+
+          if [ -s remove-package-versions.txt ]; then
+              while LANG=C IFS= read -r package_version ; do
+                  anaconda --token ${{ secrets.ANACONDA_ORG_UPLOAD_TOKEN }} remove \
+                    --force \
+                    "scipy-wheels-nightly/matplotlib/${package_version}"
```

Comment:
While @tacaswell is (AFAIK) the only person to have removed package versions before with this API (c.f. https://github.com/matplotlib/matplotlib/issues/22757#issuecomment-1142696889), this is the correct syntax to remove all wheels for a given dev release on the package index given that from https://github.com/Anaconda-Platform/anaconda-client/blob/be1e14936a8e947da94d026c990715f0596d7043/binstar_client/commands/remove.py we can see that `Package written as <user>[/<package>[/<version>[/<filename>]]]` and [for `version`](https://github.com/Anaconda-Platform/anaconda-client/blob/be1e14936a8e947da94d026c990715f0596d7043/binstar_client/commands/remove.py#L30-L31) there is the question (when run sans `--force`) of `Are you sure you want to remove the package release %s ? (and all files under it?)`.

Also you can just try this and note the same clarifying prompt

```console
$ anaconda remove scipy-wheels-nightly/matplotlib/3.6.0.dev2553+g3245d395d9
Using Anaconda API: https://api.anaconda.org
Are you sure you want to remove the package release scipy-wheels-nightly/matplotlib/3.6.0.dev2553+g3245d395d9 ? (and all files under it?) [y|N]:
```

Reply:
I have been removing things via mind-less clicking on the website (which is how I took out _all_ of the uploads at one point).

Reply:
I "tested" this by running the commands locally with the token (had to trim to 3 as we only and 4 release up!) and it deleted files as expected.